extensions/interactions/FractionInput/directives/fraction_input_interaction_directive.html

```diff
@@ -1,9 +1,11 @@
 <form ng-disabled="FractionInputForm.$invalid" ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
   <!-- This ng-form is needed for submitting answer on hitting enter. -->
-  <ng-form name="FractionInputForm">
-    <input type="text" name="answer" ng-model="answer" class="form-control"
-           ng-attr-placeholder="<[customPlaceholder !== '' ? customPlaceholder : (allowNonzeroIntegerPart ? ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER' | translate) : ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER_NO_INTEGER' | translate))]>">
-  </ng-form>
+  <fraction-input-interaction>
+    <ng-form name="FractionInputForm" class="input">
+      <input type="text" name="answer" ng-model="answer" class="form-control"
+            ng-attr-placeholder="<[customPlaceholder !== '' ? customPlaceholder : (allowNonzeroIntegerPart ? ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER' | translate) : ('I18N_INTERACTIONS_FRACTIONS_INPUT_PLACEHOLDER_NO_INTEGER' | translate))]>">
```

Comment:
Indent so that the left edge of ng-attr-placeholder matches the left edge of "type" on the previous line.

Reply:
Done